marimbabot_simulation/src/play_notes.py

```diff
@@ -0,0 +1,97 @@
+# Plays sounds for the notes published to sim_notes
+
+#!/usr/bin/env python
+
+import os
+import subprocess
+
+import librosa
+import rospy
+import sounddevice as sd
+from std_msgs.msg import String
+
+NOTES = [
+  "a4",
+  "a5",
+  "a6",
+  "ais4",
+  "ais5",
+  "ais6",
+  "b4",
+  "b5",
+  "b6",
+  "c4",
+  "c5",
+  "c6",
+  "c7",
+  "cis4",
+  "cis5",
+  "cis6",
+  "d4",
+  "d5",
+  "d6",
+  "dis4",
+  "dis5",
+  "dis6",
+  "e4",
+  "e5",
+  "e6",
+  "f4",
+  "f5",
+  "f6",
+  "fis4",
+  "fis5",
+  "fis6",
+  "g4",
+  "g5",
+  "g6",
+  "gis4",
+  "gis5",
+  "gis6",
+]
+
+
+def find_file_directory(filename):
+    home_directory = os.path.expanduser("~")
+    command = f"find {home_directory} -type f -name {filename} -exec dirname {{}} \\; -quit 2>/dev/null"
+    try:
+        output = subprocess.check_output(command, shell=True, universal_newlines=True)
```

Comment:
why don't you use Python for the whole code (os.walk etc.) or this line of code?
result = os.popen(command).read().strip()

Just out of interest

Reply:
Habit. It's also faster (probably)